src/OmniSharp/Api/v1/Navigation/OmnisharpController.FindUsages.cs

```diff
@@ -15,41 +15,48 @@ public partial class OmnisharpController
         [HttpPost("findusages")]
         public async Task<QuickFixResponse> FindUsages(FindUsagesRequest request)
         {
-            var document = _workspace.GetDocument(request.FileName);
```

Comment:
This was an odd duck... I was getting a new exception, it appears to be related to something in Roslyn itself.  It fails on both the current version of the server and the PR, I'll roll this back.